app/scripts/controllers/eventRegistrations.js

```diff
@@ -232,7 +232,7 @@ angular.module('confRegistrationWebApp')
           var registrantIndex = _.findIndex($scope.registrations[index].registrants, { 'id': registrantData.id });
           $scope.registrations[index].registrants[registrantIndex] = registrantData;
         }).error(function(){
-          modalMessage.error('Error: registrant data could be be retrieved.');
+          modalMessage.error('Error: registrant data could not be retrieved.');
```

Comment:
I suppose the only times an error would be returned would be if the user has inadequate permissions, but the endpoint requires READ, so that would be rare.  If there was an error in the API, this error is just as good if not better.

I noticed the typo when I was deploying the API locally, so it was temporarily unavailable.  I clicked the edit registrant button and the error message came up.